fuel-tx/src/tests/valid_cases/transaction/blob.rs

```diff
@@ -0,0 +1,629 @@
+#![allow(clippy::cast_possible_truncation)]
+#![allow(non_snake_case)]
+
+use super::*;
```

Comment:
I don't see the test for the case when blob id mismatch

Reply:
6fcafd41